core/src/main/java/com/segment/analytics/kotlin/core/platform/plugins/SegmentDestination.kt

```diff
@@ -101,6 +101,7 @@ class SegmentDestination : DestinationPlugin() {
     }
 
     override fun update(settings: Settings, type: Plugin.UpdateType) {
+        super.update(settings, type)
         if (settings.hasIntegrationSettings(this)) {
```

Comment:
since we're calling super, can we reuse `enabled` here?

Reply:
ideally yes, but this is more of a check for the presence of settings, which could be none in the case of manually enabled destinations